tests/stream/test_stream_smoke/0013_changelog_stream13.yaml

```diff
@@ -294,9 +295,9 @@ tests:
           from v_12180413 settings checkpoint_interval=1;
       - client: python
         query_type: table
-        depends_on_stream: v_12180413
+        depends_on_stream: 12180413213
```

Comment:
is this test flaky? why need change here..?

Reply:
```suggestion
        depends_on: 12180413213
```

Reply:
> is this test flaky? why need change here..?

yes, it's flaky one